src/net_structrw.c

```diff
@@ -116,7 +116,7 @@ boolean NET_ReadSettings(net_packet_t *packet, net_gamesettings_t *settings)
         return false;
     }
 
-    for (i = 0; i < settings->num_players; ++i)
+    for (i = 0; i < settings->num_players && i < NET_MAXPLAYERS; ++i)
```

Comment:
I'd rather use 
```
i < MIN(settings->num_players, NET_MAXPLAYERS)
```
or something like that. Do we have a `MIN` macro?

This will silently ignore the value if it's too large, what about the rest of the packet? Would something else go wrong later? Is there any valid server which might report `num_players` larger than `NET_MAXPLAYERS`?

Reply:
As stated in the bug report, I would have preferred a different approach, but the one I eventually applied is already used multiple times elsewhere in the code.